gin-sample/main.go

```diff
@@ -15,10 +16,10 @@ var (
 
 func main() {
 	logtoConfig := &client.LogtoConfig{
-		Endpoint:  "<your-logto-endpoint>", // E.g. "http://localhost:3001"
-		AppId:     "<your-app-id>",
-		AppSecret: "<your-app-secret>",
-		Resources: []string{"your-resource"}, // E.g. "https://default.logto.app/api"
+		Endpoint:  os.Getenv("ENDPOINT"),
```

Comment:
The example config needs to be reserved, since some users will confuse about it.

Reply:
Moved examples into a .env.example file for easy use